gammapy/estimators/tests/test_flux_point_estimator.py

```diff
@@ -433,29 +433,29 @@ def test_run_pwl_parameter_range(fpe_pwl):
     fp = fpe.run(datasets)
     table_no_bounds = fp.to_table()
 
-    pl.amplitude.min=0
-    pl.amplitude.max=1e-8
+    pl.amplitude.min = 0
+    pl.amplitude.max = 1e-12
 
     fp = fpe.run(datasets)
     table_with_bounds = fp.to_table()
 
     actual = table_with_bounds["norm"].data
-    assert_allclose(actual, [3.215947e-02, 3.939055e-02, 5.551115e-09], rtol=1e-3)
+    assert_allclose(actual, [0., 0., 0.], atol=1e-2)
```

Comment:
I decide to use a more stable check here, just testing whether the norm is consistent with zero.

Reply:
Looks good.